sros2/test/test_policy_to_permissions.py

```diff
@@ -23,6 +24,20 @@
 )
 
 
+if 'XML_CATALOG_FILES' not in os.environ:
```

Comment:
Could we just do this in the `__init__.py` file when you import sros2?  That seems like a hack to me, but I've been known to be too conservative about what's a 'hack' and what's a 'feature working as intended'